internal/k8shandler/indexmanagement/index_management.go

```diff
@@ -113,10 +142,16 @@ func newPolicySpec(name string, maxIndexAge esapi.TimeUnit, hotPhaseAge esapi.Ti
 				},
 			},
 			Delete: &esapi.IndexManagementDeletePhaseSpec{
-				MinAge: maxIndexAge,
+				MinAge:                  retentionPolicy.MaxAge,
```

Comment:
MinAge assigned from MaxAge. (deleted line also doing same though)

Reply:
Deleted line was doing same but I wanted more variables which is part of this `retentionPolicy` variable. I made the function definition generic and removed the individual variable.